### ISSUE TITLE
[Support Question] Boron & Lithium in Mining Dimension

### ISSUE THREAD

**Issue by Morpheus1101** [turbodiesel4598/NuclearCraft#268]
we've recently added Nuclearcraft to the MMD Public Server pack and even though a reset of Aroma's Mining Dimension was done, ive discorerd after quarrying 70+ chunks that neither Boron or Lithium have been generated in that dimension even though it is whitelisted, is this something on this mods end or Aroma's mod end ?

**Comment by turbodiesel4598** [comment]
Hmm, should be working - works in FTB Revelation, after all. How about the other NC ores?

**Comment by Morpheus1101** [comment]
Im not sure on the others tbh as we have another mod providing them, but certainly the Beryllium, Sulfur, Rhodochrosite & Fluorite are present

**Comment by turbodiesel4598** [comment]
What is the mining dimension's ID?

**Comment by Morpheus1101** [comment]
6

**Comment by turbodiesel4598** [comment]
Are the ores generating in your overworld correctly? If so, I'm very confused as to why they are generating there, but not the mining world...

**Comment by Morpheus1101** [comment]
Yes, i went to a distance where i know hasnt been loaded yet an i was getting the ores aok in overworld, which is what had me confused as well to why they where working overworld and no tin mining dimension.  I havent tested out DeepDark as yet either, to see if its spawning there

**Comment by Deanosim** [comment]
The mining dimension if -6 not 6
You can clearly see it in the configs for Aroma1997's Mining Dimension.

**Comment by WillWill56** [comment]
As stated above, this is a simple fix of changing a positive number to a negative number. Aroma's mining dimension generates by default in a folder called DIM-6, which stands for "dimension negative six". I'm surprised this wasn't picked up sooner...
